test/jtreg-ext/requires/VMProps.java

```diff
@@ -138,6 +138,7 @@ public Map<String, String> call() {
         map.put("jdk.containerized", this::jdkContainerized);
         map.put("vm.flagless", this::isFlagless);
         map.put("jdk.foreign.linker", this::jdkForeignLinker);
+        map.put("vm.cgroup.tools", this::cgroupTools);
```

Comment:
Why the `vm` prefix? Could be just `cgroup.tools` no? This isn't a JVM property.